### DISCUSSION TITLE
How to ignore some type of errors?

### DISCUSSION THREAD

**Discussion by marcofarnezi** [phpro/grumphp#941] [Q&A]
Id like to ignore some types of errors. Has grumphp a list of false-positives? If yes, has someone the documentation?

**Comment by veewee** [comment]
We don't have that in GrumPHP.
Since we rely on external tools in most cases, it should be marked as a false positive inside the tools you are using. 

Does this answer your question? If not, feel free to elaborate on the specific situation you are thinking of
